WirelessModules/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
That's a valid question actualy. There is merit in adding a license to a private repo since it then provides some sort of license on the code that is put on the code in this repository. Since MHP doesn't require us to write any NDA or anything.

Instead of an MIT license, are there any licenses that we should look into @hallgchris? See https://choosealicense.com/